.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,21 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
```

Comment:
nit: is it possible to change myName to something more meaningful?

Reply:
This file is generated by IntelliJ, I tried to rename the config (which changes "Project Default") but it creates other files so I prefer to keep it as simple as possible. If you feel strongly about it, I will try to figure how this can be fixed.

Reply:
ah, it's ok then.